frontend/src/app/drive/global/page.tsx

```diff
@@ -4,7 +4,7 @@ import { FileActionButtons, FileTable } from "@/components/common/FileTable";
 import { FileDropZone } from "@/components/Files/FileDropZone";
 import { NoUploadsPlaceholder } from "@/components/Files/NoUploadsPlaceholder";
 import { ApiService } from "@/services/api";
-import { useCallback, useEffect, useState } from "react";
+import { Fragment, useCallback, useEffect, useState } from "react";
```

Comment:
Fragment is imported but unused, furthermore, Fragment is not needed anymore since react 16.2, with the introduction of <> </>
These non-breaking errors could have been catch earlier if eslint was properly setup, this is what this line refer to in the build process 
![image](https://github.com/user-attachments/assets/cd46fad5-fab0-4d07-9bbb-1645142c73f0)

Reply:
I personally prefer using Fragment because visually <></> is a bit hard to read

Reply:
Anyways, the eslint PR is to solve this one